O365-iOS-Microsoft-Graph-Connect-Swift/AuthenticationClass.swift

```diff
@@ -30,12 +30,15 @@ class AuthenticationClass {
             }
 
             let kClientId = String(redirectUrl[msalRange.upperBound...])
+            let authority = try MSALAADAuthority(url: URL(string:ApplicationConstants.kAuthority)!)
+            
+            // Redirect uri will be constucted automatically in the form of "msal<your-client-id-here>://auth" if not provided.
```

Comment:
Let's update to msauth.bundleId format so that broker could be used

Reply:
broker support added :)